### PR TITLE
Ch1 fig cleanup

### PR DESCRIPTION
**Pull request by lheagy**
PR for the code annotation figures in Chapter 1. I also included the ppt where I made the annotations so updates can be made in the future. I also made a couple of minor updates to have the large tables scroll

There is one lingering note for @joelostblom 
image

### CODE REVIEW

File: source/intro.md [hunk 1]
```diff
@@ -324,6 +322,7 @@ Syntax for the `read_csv` function
 
 +++
 ```{code-cell} ipython3
+:tags: ["output_scroll"]
```
Comment (lheagy) [inline source/intro.md:325]: this is handy for large tables

**Review by joelostblom — COMMENTED** [review]
I think this looks good overall! What do you think about removing the assignments in the three figures `altair_syntax`, `filter_rows_and_columns`,  and `sort_values`? That will give more real estate for the important part of those images (the explaining of the different parts of the commands)

**Comment by joelostblom** [comment]
I added my section in https://github.com/UBC-DSCI/introduction-to-datascience-python/pull/101

**Comment by trevorcampbell** [comment]
> I think this looks good overall! What do you think about removing the assignments in the three figures altair_syntax, filter_rows_and_columns, and sort_values? That will give more real estate for the important part of those images (the explaining of the different parts of the commands)

I don't think that's worth the edit honestly ;)

This looks good to me aside from the `~$blah.pptx` file that got committed. I'll remove that and merge.

**Pull request merged**